.gitignore

```diff
@@ -36,7 +37,6 @@
 build_*.err
 build_*.log
 build_*.wrn
-Build/.vs/
```

Comment:
I think it's redundant since `.vs/` doesn't require that the folder be in the root, if it appears anywhere in the folder structure it will be ignored.